Developers/Writing_Bears.rst

```diff
@@ -276,3 +276,148 @@ parameter in ``HiddenResult`` should be the instance of the Bear that yields
 this result (in this case ``self``), and second argument should be the content
 we want to transfer between the Bears. Here we use a list of strings as content
 but it can be any object.
+
+More Configuration Options
+--------------------------
+coala provides settings to further configure your bear according to your needs.
+Here is the list of all the settings you can use:
+
+- `LANGUAGES`_
+- `REQUIREMENTS`_
+- `INCLUDE_LOCAL_FILES`_
+- `CAN_DETECT and CAN_FIX`_
+- `BEAR_DEPS`_
+- `Other Settings`_
+
+
+LANGUAGES
+~~~~~~~~~
+
+To indicate which languages your bear supports,
+you need to give it a `set` of strings as a value:
+
+.. code:: python
+
+    class SomeBear(Bear):
+        LANGUAGES = {'C', 'CPP','C#', 'D'}
+
+REQUIREMENTS
+~~~~~~~~~~~~
+
+To indicate the requirements of the bear,
+assign ``REQUIREMENTS`` a set with instances of 
+subclass of ``PackageRequirement`` such as:
+
+- PipRequirement
+- NpmRequirement
+- CondaRequirement
+- DistributionRequirement
+- GemRequirement
+- GoRequirement
+- JuliaRequirement
+- RscriptRequirement
+
+.. code:: python
+
+    class SomeBear(Bear):
+        REQUIREMENTS = {
+        PipRequirement('coala_decorators', '0.2.1')}
+
+To specify multiple requirements you can use
+the multiple method. This can receive both tuples of strings,
+in case you want a specific version, or a simple string,
+in case you want the latest version to be specified.
+
+.. code:: python
+
+    class SomeBear(Bear):
+        REQUIREMENTS = PipRequirement.multiple(
+            ('colorama', '0.1'),
+            'coala_decorators')
+
+INCLUDE_LOCAL_FILES
+~~~~~~~~~~~~~~~~~~~
+
+If your bear needs to include
+local files, then specify it by giving strings
+containing relative file paths to the
+``INCLUDE_LOCAL_FILES``.
+
+.. code:: python
+
+    class SomeBear(Bear):
+        INCLUDE_LOCAL_FILES = {'checkstyle.jar',
+            'google_checks.xml'}
```

Comment:
I did a `git grep INCLUDE_LOCAL_FILES` on the repo and it seems like this not being used anywhere.